app/views/administrate/application/index.html.erb

```diff
@@ -21,7 +21,9 @@ It renders the `_table` partial to display details about the resources.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<% content_for(:title) { page.resource_name.pluralize.titleize } %>
+<% content_for(:title) do
+  page.resource_name.classify.constantize.model_name.human(default: page.resource_name.pluralize.titleize)
```

Comment:
Maybe the pluralization should happen after the `human(...)` call? That way it would apply even if it didn't use the default.